### PR TITLE
Polish maintenance windows

### PR DESCRIPTION
**Pull request by alexec**
Screen Shot 2019-10-02 at 11 25 53
Screen Shot 2019-10-02 at 11 20 44
Screen Shot 2019-10-02 at 11 07 53

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -56,6 +56,7 @@ nav:
     - user-guide/resource_hooks.md
     - user-guide/selective_sync.md
     - user-guide/sync-waves.md
+    - user-guide/maintenance_windows.md
```
Comment (alexec) [inline mkdocs.yml]: docs must be listed in this file

File: ui/src/app/applications/components/utils.tsx [hunk 2]
```diff
@@ -374,17 +374,16 @@ export const MaintenanceWindowStatusIcon = ({state, window}: { state: appModels.
...
         className = 'fa fa-stop-circle';
-        color = COLORS.maintenance_state.active;
-    } else {
+    } else if (state.windows.length > 0) {
         className = 'fa fa-stop-circle';
-        color = COLORS.maintenance_state.inactive;
     }
-    return <React.Fragment><i title={name} className={className} style={{color}}/>{name}</React.Fragment>;
+    return (
+        <a href={`/settings/projects/${project}?tab=maintenance`} className={state.windows ? 'warning' : ''}>
```
Comment (alexec) [inline ui/src/app/applications/components/utils.tsx]: deep link to project

File: ui/src/app/applications/components/utils.tsx [hunk 3]
```diff
@@ -374,17 +374,16 @@ export const MaintenanceWindowStatusIcon = ({state, window}: { state: appModels.
...
 };
 
-export const ApplicationMaintenanceWindowStatusIcon = ({state}: { state: appModels.ApplicationMaintenanceState }) => {
-    let className = 'fa fa-question-circle';
-    let color = COLORS.maintenance_state.unknown;
-    const name = 'Maintenance';
-
-    if (state.windows !== undefined && state.windows.length > 0) {
+export const ApplicationMaintenanceWindowStatusIcon = ({project, state}: { project: string, state: appModels.ApplicationMaintenanceState }) => {
+    let className = 'fa fa-stop-circle';
```
Comment (alexec) [inline ui/src/app/applications/components/utils.tsx]: i had trouble understanding the logic here, the `question-circle` only appears in an edge case, wonder if this is is bug or not

**Comment by alexec** [comment]
@adamjohnson01 FYI

File: ui/src/app/applications/components/utils.tsx (hunk 3 above)
Comment (adamjohnson01) [reply ui/src/app/applications/components/utils.tsx]: I think that I left it there in case the state could not be looked up.

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**